frontend/tsconfig.json

```diff
@@ -12,8 +12,8 @@
     "strict": true,
     "sourceMap": true,
     "inlineSources": true,
-    "importsNotUsedAsValues": "error",
+    "verbatimModuleSyntax": true,
     "useUnknownInCatchVariables": true
   },
-  "include": ["packages/**"]
+  "include": ["packages/*/src"]
```

Comment:
I guess, we shouldn't include anything related to `packages` here because they have their own `tsconfig.json`. This part should include only TS and JS files related to scripts or other non-production code.

Reply:
True, but ESLint TypeScript plugin gets with such `tsconfig.json`. Any suggestions for how to make it work otherwise?

Reply:
(Hopefully) figured it out.